src/App.tsx

```diff
@@ -73,7 +76,7 @@ export const App = () => {
             value={title}
             onChange={event => setTitle(event.target.value)}
           />
-          {(!title && hasError) && (
+          {(!title && hasTitleError) && (
```

Comment:
Don't check same condition twice. Use only error state here.

Reply:
without "!title" code doesn't work correctly(

Reply:
@rstarovoit Problem is in your submit handler